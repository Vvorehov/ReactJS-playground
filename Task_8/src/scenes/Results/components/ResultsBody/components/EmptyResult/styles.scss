.empty-result {
  display: flex;
  align-items: center;
  justify-content: center;
  min-height: 60vh;
  font-size: 4.8rem;
  font-weight: 700;
}