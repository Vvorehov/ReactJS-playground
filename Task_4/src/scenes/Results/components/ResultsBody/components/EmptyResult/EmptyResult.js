import React, {Component} from 'react';
import { render } from "react-dom";
import './styles.scss'

class EmptyResult extends Component {

  render() {
    return (
        <div className="empty-result">No films found</div>
    );
  }
}

export default EmptyResult;