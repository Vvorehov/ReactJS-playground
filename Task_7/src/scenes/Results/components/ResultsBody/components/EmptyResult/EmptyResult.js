import React, {Component} from 'react';
import { render } from "react-dom";
import './styles.scss'

class EmptyResult extends Component {

  render() {
    return (
        <div className="empty-result">{this.props.text}</div>
    );
  }
}

export default EmptyResult;