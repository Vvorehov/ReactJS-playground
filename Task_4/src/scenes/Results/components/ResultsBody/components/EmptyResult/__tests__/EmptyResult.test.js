import React from "react";
import EmptyResult from "../EmptyResult";
import TestRenderer from 'react-test-renderer';

describe('EmptyResult layout complete', () => {

    it('ResultsBody Render correct EmptyResult', () => {

        const component = TestRenderer.create(
            <EmptyResult />
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

