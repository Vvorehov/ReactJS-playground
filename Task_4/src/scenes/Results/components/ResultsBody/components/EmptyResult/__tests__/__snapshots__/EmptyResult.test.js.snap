// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyResult layout complete ResultsBody Render correct EmptyResult 1`] = `
<div
  className="empty-result"
>
  No films found
</div>
`;
